batchflow/models/tf/nn/train.py

```diff
@@ -0,0 +1,74 @@
+""" Helpers for training """
+from math import pi
+
+import tensorflow as tf
+from tensorflow.python.framework import ops # pylint: disable=no-name-in-module
+from tensorflow.python.ops import math_ops # pylint: disable=no-name-in-module
+from tensorflow.math import sin, asin, floor # pylint: disable=import-error
+
+
+def piecewise_constant(global_step, *args, **kwargs):
+    """ Constant learning rate decay (uses global_step param instead of x) """
+    return tf.train.piecewise_constant(global_step, *args, **kwargs)
+
+def cyclic_learning_rate(learning_rate, global_step, max_lr=0.1, step_size=10, mode='triangular', name=None):
+    """Applies cyclic learning rate (CLR).
+    https://arxiv.org/pdf/1506.01186.pdf.
+
+    This function varies the learning rate between the
+    minimum (learning_rate) and the maximum (max_lr).
+    It returns the cyclic learning rate.
+
+    Args:
+    learning_rate: A scalar `float32` or `float64` `Tensor` or a
+        Python number.  The initial learning rate which is the lower bound
```

Comment:
here you write it's initial learning rate, but in `mode` above there are different initial values/ please clarify